src/compass/schemas/s1_cslc_geo.yaml

```diff
@@ -127,6 +127,10 @@ rdr2geo_options:
     compute_local_incidence_angle: bool(required=False)
     # Enable/disable azimuth angle output
     compute_azimuth_angle: bool(required=False)
+    # Enable/disable ground to satellite East LOS vector
+    compute_ground_to_sat_east: bool(required=False)
+    # Enable/disable ground to satellite North LOS vector
+    compute_ground_to_sat_north: bool(required=False)
```

Comment:
These need to be added to defaults.

Reply:
They have been already added to the defaults. Please, double-check.